hedera-node/src/main/java/com/hedera/services/txns/validation/ContextOptionValidator.java

```diff
@@ -131,7 +131,9 @@ public ResponseCodeEnum tokenTransfersLengthCheck(List<TokenTransferList> tokenT
 		int count = 0;
 		for (var tokenTransferList : tokenTransferLists) {
 			int transferCounts = tokenTransferList.getTransfersCount();
-			if (transferCounts == 0) {
+			int nftTransferCount = tokenTransferList.getNftTransfersCount();
+
+			if (transferCounts == 0 && nftTransferCount == 0) {
```

Comment:
`EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS` is used only for FUNGIBLES and it refers to the `account amount` properties in the transfer list. For non-fungibles we do not have account amounts and it will be confusing. I will add this for discussion in our weekly sync call

Reply:
It has been decided to stay as it is developed on the weekly cadence call